tt_metal/jit_build/data_format.cpp

```diff
@@ -326,44 +230,44 @@ const DataFormat get_single_pack_src_format(
             default: pack_src_format = DataFormat::Lf8; break;
         }
     } else if (input_format == DataFormat::UInt16) {
-        pack_src_format = output_format;
+        pack_src_format = input_format;
```

Comment:
Why is the usage changed from output_format to input_format?

How big is the blast radius?

Reply:
We are no longer making a distinction between input and output.  Really the variable should have changed.

Reply:
Changing the variable name.